MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include README.md
+include LICENSE
```

Comment:
why is this needed? the metadata already says which license it is, we don't need to include the full text surely?

Reply:
https://www.gnu.org/licenses/gpl-howto.html
says
"You should also include a copy of the license itself somewhere in the distribution of your program. All programs, whether they are released under the GPL or LGPL, should include the text version of the GPL."

I do believe that the packaged version of charmcraft counts as the "distribution" of the program, and thus it *should* include the full text of the LICENSE that it is using. (GNU also happens to use COPYING rather than LICENSE, but I don't particularly care about the name as much as 'should you include a copy' seems to bias that it should be part of the program.)

Reply:
we're apache-licensed, but indeed it also says

>      (a) You must give any other recipients of the Work or
>          Derivative Works a copy of this License; and

Reply:
Perfect, so we need this.